### ISSUE TITLE
Kitty installation on Lenovo Duet 3 (aarch64) ChromeOs incompatible

### ISSUE THREAD

**Issue by niratschi** [cbeley/beleyenv#34]
Hey,

I tried to install just kitty on my Lenovo Duet 3 with your script. The script downloads kitty in Version 0.27.1, but this version is incompatible with debian 11. It requires a newer glibc version of 2.33/2.34 while debian is on 2.31.

I could install kitty via apt-get install kitty, but now i have already installed the newer version via the script. Could you give me a hint, why the installer.sh did not autodetect the right version for debian 11 or how to remove the installed 0.27.1 version?

**Comment by cbeley** [comment]
This is likely an issue with your local Linux container. I don't use ChromeOS as much anymore, but I went ahead and ran `./installScripts/installKitty/index.sh`.

I'm now on Kitty 0.27.1 and all is well.

`cat /etc/os-release` tells me I'm on Debian 11.

`ldd --version ldd` confirms I am using `Glibc 2.31-13+deb11u5`.

Have you tried running the script on a freshly created linux container? If so, and you're still seeing the issue, I'd be curious to debug further. If not, you're unfortunately on your own.

**Comment by cbeley** [comment]
I also just went ahead and did a `sudo apt-get update && sudo apt-get dist-upgrade && sudo apt-get autoremove` as a sanity check. All still well.

**Comment by niratschi** [comment]
Thank you for your fast answer, btw its awesome, that you share all this!
I am on a fresh chromebook, only thing i did before installing kitty, was installing vs code and docker. 
When i try to start kitty, the following error is shown:

```
kitty: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by kitty)
kitty: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by kitty)
```

I will try again tonight with a fresh linux container.

**Comment by cbeley** [comment]
Hmm, you know what it could be? Duet 3 is ARM, and I use a Galaxy Chromebook, which is x86.

My script in beleyenv doesn't do much more than handle the creation of the icon for you. Otherwise, it is just using the official kitty install script. It's possible his install script doesn't support this combo of glibc and ARM.

Knowing him, he'd tell you to go compile from source and that your OS is too old...

**Comment by cbeley** [comment]
Maybe see if you can run his static build: https://github.com/kovidgoyal/kitty/releases/download/v0.27.1/kitten-linux-arm64 .

**Comment by niratschi** [comment]
I have deleted kitty and tried to run it via 

1. apt-get install kitty (its an old version 19.xx)
2. the static build you linked

The old kitty version (1) looks totaly like a plain terminal. It sems to be not compatible with my setup.

The static file you linked is not the terminal itself, its a static standalone for unix server to remote controll kitty. I tried to reinstall kitty v0.27.1 using the kitty-0.27.0-arm64.txz file from the release link, but it has the same error with the glibc version.

I think the system i have is not supported
